### PR TITLE
Just a group of people who know other people

### PR DESCRIPTION
**Pull request by maxhipperson**
We didn't use any lists; we used nested dictionaries.

@donalmclaughlin
Answers UCL-RITS/rsd-classwork-2019#8

### CODE REVIEW

File: group.py [hunk 1]
```diff
@@ -2,4 +2,47 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+    'person': {
+        'age': '',
+        'job': '',
+        'connections': {
+            'connections_name': 'connection_type'
```
Comment (dyylan) [inline group.py]: A list for the connection type would allow multiple connections

**Review by dyylan — COMMENTED** [review]
Looks good

File: group.py (hunk 1 above)
Comment (maxhipperson) [reply group.py]: Ah yeah, did think about this after the pull request

**Comment by maxhipperson** [comment]
@Dyylan added lists, should be good now

**Pull request closed without merging**